helix-core/src/main/java/org/apache/helix/controller/GenericHelixController.java

```diff
@@ -235,7 +235,9 @@ public static GenericHelixController getLeaderController(String clusterName) {
     if (clusterName != null) {
       ImmutableSet<GenericHelixController> controllers = _helixControllerFactory.get(clusterName);
       if (controllers != null) {
-        return controllers.stream().filter(controller -> controller._helixManager.isLeader())
+        return controllers.stream()
+            .filter(controller -> controller._helixManager != null)
+            .filter(controller -> controller._helixManager.isLeader())
             .findAny().orElse(null);
```

Comment:
Are you sure using Helix format???

Reply:
Sorry, not sure I understand what you mean with "helix format". Is the formatting of the lines somehow faulty? From what I see the ".filter(...)" lines are indented the same way as the ".findAny()" line was before, so I don't get what is wrong, sorry.

Reply:
Ok, I now understood what you meant. I applied the format to all the files I touched. I don't necessarily find it more readable, but then I guess this is also a question of taste. Hopefully this is ok now.

Reply:
Please make sure you load the Helix intellij / eclipse auto code formatter.

Reply:
Done. reverted global reformatting and only applied it on the lines I had changed initially.